gradle/libs.versions.toml

```diff
@@ -35,7 +35,7 @@ agp = "7.3.1"
 twisterrob = "0.15-SNAPSHOT"
 
 # https://github.com/JetBrains/kotlin/blob/master/ChangeLog.md
-kotlin = "1.7.22"
+kotlin = "1.8.0"
```

Comment:
Lint failures look like false positives, report and suppress. not sure how it's related to this PR 🫣